k8s/deployments/mpm-backend.yaml

```diff
@@ -30,39 +30,39 @@ spec:
                   key: APP_KEY
             - name: APP_DEBUG
               value: "false"
-
             - name: MPM_LOAD_DEMO_DATA
               value: "true"
-
             - name: DB_CONNECTION
               value: micro_power_manager
             - name: DB_HOST
-              value: db
+              valueFrom:
+                secretKeyRef:
+                  name: mpm-secrets
+                  key: MYSQL_HOST
```

Comment:
Can you please name this variable `DB_HOST` for consistency? `MYSQL_HOST` was used in the MariaDB container, which no longer exists. Also adapt the secrets file.